notebooks/Analyze_user_behavior_across_different_lending_protocols.ipynb

```diff
@@ -678,6 +975,14 @@
     "plot_venn_diagram_per_token(collateral_amounts, 'USDC', 'Total Collateral USD Amounts for USDC Across Top 3 Protocols')\n",
```

Comment:
There must be some error because there's definitely an overlap for USDC collateral and debt. Pls check it.